manifest_staging/charts/aad-pod-identity/values.yaml

```diff
@@ -163,7 +163,7 @@ mic:
 
 nmi:
   image: nmi
-  tag: v1.8.6
+  tag: v1.8.7
```

Comment:
You don't need to bump the version to v1.8.6here. We will bump it in a separate release PR (e.g. https://github.com/Azure/aad-pod-identity/pull/1213)

Reply:
ok
changing